app/views/shared/_loading.html.haml

```diff
@@ -1,14 +1 @@
 #loading-container
-
-      
-:scss
-  #loading-container {
-    display: none;
-    left:40%;
-    position: absolute;
-    text-align: center;
-    background: white;
-    padding: 10px;
-    font-size: large;
-    z-index: 100;
-  }
```

Comment:
After the upgrade, this :scss instance was causing the error:

```
Failure/Error: <%= render :partial => 'shared/loading' %>
ActionView::Template::Error: wrong number of arguments (given 3, expected 1)
```

Removing it from here and adding the new _loading.scss file fixed that.